src/features/email/lib/project-invitation.ts

```diff
@@ -0,0 +1,71 @@
+import { createTransport } from "nodemailer";
+import { Resend } from "resend";
+
+import { projectInvitationTemplate } from "@/src/features/email/templates/ProjectInvitation";
+
+export const sendProjectInvitation = async (
+  to: string,
+  senderName: string,
+  projectName: string,
+) => {
+  try {
```

Comment:
Check if all required envs to send emails are set, if not, console.error and return. Do not throw an error, some users will not set up email sending.

Reply:
-> No need to have defaults anymore with ??